app/assets/stylesheets/components/_service-comparison.scss

```diff
@@ -0,0 +1,204 @@
+.service-comparison {
+  box-sizing: border-box;
+  h2.title {
+    text-align: center;
+    margin-bottom: 5px;
+  }
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  .comparison-component {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    border: 1px solid $color-grey-medium;
+    border-radius: 10px;
+    padding: 15px;
+  }
+
+  .service-set {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    font-size: 1.6rem;
+
+    @media screen and (min-width: $site-max-up) {
+      flex-direction: row;
+      align-items: center;
+      justify-content: center;
+    }
+
+    &--hr {
+      @media screen and (min-width: $site-max-up) {
+        border-bottom: 1px solid $color-grey-medium;
+      }
+    }
+
+    &--mobile-only {
+      @media screen and (min-width: $site-max-up) {
+        display: none;
+      }
+    }
+
+    &--mobile-excluded {
+      @media screen and (max-width: $site-max-up) {
+        display: none !important;
+      }
+    }
+
+    &__header {
+      display: flex;
+      flex-direction: column;
+      margin-left: 1rem;
+
+      @media screen and (min-width: $site-max-up) {
+        width: 222px;
+        justify-content: flex-end;
+        align-items: flex-end;
+      }
+
+      &--limited {
+        @media screen and (min-width: $site-max-up) {
+          padding-left: 3.5rem;
+        }
+      }
+    }
+
+    &__title {
+      font-weight: 700;
+      padding: .5rem 0;
+      &--normal {
+        font-weight: normal;
+        @media screen and (min-width: $site-max-up) {
+          text-align: right;
+        }
+      }
+    }
+
+    &__subtitle {
+      font-style: italic;
+      display: flex;
+      font-size: 1.4rem;
+      line-height: 1.9rem;
+      padding-bottom: .5rem;
+      @media screen and (min-width: $site-max-up) {
+        text-align: right;
+      }
+      &--fixie {
```

Comment:
fixie is a fun name, but I wasn't really sure what it meant (I was expecting a [clearfix](https://css-tricks.com/snippets/css/clear-fix/)).